style.css

```diff
@@ -51,6 +51,12 @@ body, html {
   flex-grow: 1;
 }
 
+#connections-screen #github-startpage-link {
```

Comment:
Why do we have two different selectors for the same thing?
Since the github-link has an id, it should be possible to have only one selector for that id.